Assets/Scripts/Models/Tile.cs

```diff
@@ -112,15 +112,14 @@ public float PathfindingCost
 
     public int Y { get; private set; }
 
-    public float MovementModifier;
+    public float MovementModifier { get; private set; }
```

Comment:
We haven't done any real design on it, but I'm thinking you add the condition `Bloody` to the tile so there can be a specific job to clean it up later. But while it's set, the movement is modified for that Tile.